src/EventStream.js

```diff
@@ -90,23 +102,66 @@ class EventStream extends EventEmitter {
 		this.removeAllListeners();
 	}
 
+	/* Private methods */
+
+	emitSafe(event, param) {
```

Comment:
Avoids silently swallowing errors when someone make a typo in their event handler